sample/aidial.config.json

```diff
@@ -59,7 +59,10 @@
                     "folderAttachmentsSupported": false
                 },
                 "maxInputAttachments": 10,
-                "inputAttachmentTypes": ["type1", "type2"]
+                "inputAttachmentTypes": ["type1", "type2"],
+                "defaults": {
+                    "param1": "value"
```

Comment:
Please add tests not only for string parameters, but for floats and integers as well, because this is what we would actually need to be configured:

```json
    "defaults": {
        "max_tokens": 2000, 
        "temperature": 0.38, 
        "top_p": 0.0001,
        "frequency_penalty": 1.12
    }
```

Reply:
fixed